polaris-react/src/components/Filters/Filters.scss

```diff
@@ -58,14 +58,22 @@
 
 .hideQueryField .FiltersInner {
   flex: 1;
+  padding: var(--p-space-3);
 }
 
 @media #{$p-breakpoints-md-up} {
   .FiltersInner {
     overflow: visible;
     flex-wrap: wrap;
     gap: var(--p-space-2);
-    padding: calc(var(--p-space-2) - var(--p-space-05)) var(--p-space-2);
+    // stylelint-disable-next-line -- No 6px space token
+    padding: 0.375rem var(--p-space-2);
```

Comment:
We do have a 6px token

https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/token-groups/space.ts#L46-L48

Reply:
> fix because it was hard coded at 6px instead of rem

Note: Our postcss config [converts all px to rems](https://github.com/Shopify/polaris/blob/67070e4af31539cd61f80f3c57b91316c9b6b3c4/polaris-react/config/postcss-plugins.js#L6)